specification/notificationhubs/resource-manager/readme.md

```diff
@@ -24,6 +24,16 @@ To see additional help and options, run:
 ### Basic Information
 These are the global settings for the NotificationHubs API.
 
+directive:

```

Comment:
The directives must be defined in the yaml section for the tag containing this definition